### PR TITLE
Upgrade airseeker dependencies

### LINKED ISSUE #160: Update Airnode (and other) dependencies
There are stale PRs from dependabot updating Airnode (and other) dependencies that are failing. They require some manual changes.

### PR DESCRIPTION
**Pull request by acenolaza**
Closes  #160

I had to copy over the zod schemas from Airnode because the [zod library used there](https://github.com/api3dao/airnode/blob/v0.7.2/packages/airnode-validator/package.json#L33) is way to old and it is no longer compatible with the latest version of zod.

Another issue was that [chainOptionsSchema](https://github.com/api3dao/airnode/blob/v0.7.2/packages/airnode-validator/src/config/config.ts#L85) was changed from a `z.object` to a `z.discrimitantedUnion` [here](https://github.com/api3dao/airnode/pull/1103) and this means that methods like `merge` or `extend` are no longer available. Even though I could just have used `z.interection` to add the `gasOracle` schema the validation still failed because each option of the discriminated union was calling `strict()`.

Bottom line is that Airseeker no longer depends on zod schemas from Airnode and I believe that's a good thing :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,11 +47,11 @@
...
-    "@api3/airnode-abi": "^0.6.2",
-    "@api3/airnode-node": "^0.6.2",
-    "@api3/airnode-protocol-v1": "^0.5.0",
-    "@api3/airnode-utilities": "^0.6.2",
-    "@api3/airnode-validator": "^0.6.2",
+    "@api3/airnode-abi": "^0.7.2",
+    "@api3/airnode-node": "^0.7.2",
+    "@api3/airnode-protocol-v1": "^0.6.0",
+    "@api3/airnode-utilities": "^0.7.2",
+    "@api3/airnode-validator": "^0.7.2",
```
Comment (Siegrift) [inline package.json]: I guess we no longer need Airnode validator in dependencies.

File: src/gas-oracle.test.ts [hunk 2]
```diff
@@ -59,7 +59,7 @@ const config: Config = {
         },
       },
       options: {
-        txType: 'legacy',
+        txType: 'eip1559',
```
Comment (Siegrift) [inline src/gas-oracle.test.ts:62]: Is this correct? Previously the value was `legacy`.

**Review by Siegrift — COMMENTED** [review]
> I had to copy over the zod schemas from Airnode because the [zod library used there](https://github.com/api3dao/airnode/blob/v0.7.2/packages/airnode-validator/package.json#L33) is way to old and it is no longer compatible with the latest version of zod.

I also faced a similar issue when updating Airnode dependencies. See: https://github.com/api3dao/airnode/pull/1325#issuecomment-1191772400

> Bottom line is that Airseeker no longer depends on zod schemas from Airnode and I believe that's a good thing :)

I would not say so. Similar argument could be said about other airnode-xyz dependency, but reusing that code is simply convenient. But I support your decision by copying over the necessary validations because of the "zod" issues.

File: package.json (hunk 1 above)
Comment (acenolaza) [reply package.json]: Good catch!

File: src/gas-oracle.test.ts (hunk 2 above)
Comment (acenolaza) [reply src/gas-oracle.test.ts:62]: It used to be `legacy` but it had the properties of the `eip1559` so I assume that either was not being used in those tests or it was using the properties of `eip1559` but had the wrong type set 🤷🏻

**Comment by acenolaza** [comment]
> > I had to copy over the zod schemas from Airnode because the [zod library used there](https://github.com/api3dao/airnode/blob/v0.7.2/packages/airnode-validator/package.json#L33) is way to old and it is no longer compatible with the latest version of zod.
> 
> I also faced a similar issue when updating Airnode dependencies. See: [api3dao/airnode#1325 (comment)](https://github.com/api3dao/airnode/pull/1325#issuecomment-1191772400)
> 
> > Bottom line is that Airseeker no longer depends on zod schemas from Airnode and I believe that's a good thing :)
> 
> I would not say so. Similar argument could be said about other airnode-xyz dependency, but reusing that code is simply convenient. But I support your decision by copying over the necessary validations because of the "zod" issues.

Right 😁 I meant in this particular case

**Review by Siegrift — APPROVED** [review]
👍 LGTM
